main.js

```diff
@@ -1,42 +1,30 @@
+document.getElementById("form-button").onclick = function() {
 
-// 課題①-4
+  let f_number = document.getElementById("fizznumber").value;
+  let b_number = document.getElementById("buzznumber").value;
+  let parent = document.getElementById('answer');
+ 
 
-function AddTask(){
-
-  const tasks = [
-    {
-      content: "机を片付ける",
-      genre: "掃除",
-    },
-    {
-      content: "牛乳を買う",
-      genre: "買い物",
-    },
-    {
-      content: "散歩する",
-      genre: "運動",
+  if(isNaN(f_number) || isNaN(b_number)) {
+    var e = document.createElement("p");
+    e.innerHTML = "整数値を入力してください";
+    parent.appendChild(e);
+  } else {
+    for (var i = 1; i < 99; i++) {
+      var e = document.createElement("div");
+      if (i % f_number === 0 && i % b_number === 0) {
+        e.innerText = "FizzBuzz" + " " + i ;
+        parent.appendChild(e);
+      }
+      else if (i % f_number === 0){
+        e.innerText = "Fizz" + " " + i ;
+        parent.appendChild(e);
+      }
+      else if(i % b_number === 0){
+        e.innerText = "Buzz" + " " + i ;
+        parent.appendChild(e);
+      }
```

Comment:
コード規約はプロジェクトや会社によって変わってきますが、一般的に多いコード規約についてコメントします。
`{}`の両外側にはスペースをあけましょう。`()`の外側にスペースを空けている場合と空けていない場合がありますが、空けないように統一しましょう。 改行についても下記のように修正しましょう。
```suggestion
      if(i % f_number === 0 && i % b_number === 0) {
        e.innerText = "FizzBuzz" + " " + i ;
        parent.appendChild(e);
      } else if(i % f_number === 0) {
        e.innerText = "Fizz" + " " + i ;
        parent.appendChild(e);
      } else if(i % b_number === 0) {
        e.innerText = "Buzz" + " " + i ;
        parent.appendChild(e);
      }
```